extras/beast/unit_test/dstream.hpp

```diff
@@ -8,9 +8,7 @@
 #ifndef BEAST_UNIT_TEST_DSTREAM_HPP
 #define BEAST_UNIT_TEST_DSTREAM_HPP
 
-#include <boost/utility/base_from_member.hpp>
 #include <iostream>
-#include <memory>
 #include <ostream>
 #include <sstream>
 #include <string>
```

Comment:
Here's the set of headers I recommend:
````
#include <ios>
#include <memory>
#include <ostream>
#include <streambuf>
#include <string>
````
You could get away with less, but there is little motivation to do so.